test/python/taskbasedsimulator/benchmark.py

```diff
@@ -1,14 +1,16 @@
 import datetime
 import time
-from mqt.ddsim.pathqasmsimulator import PathQasmSimulator, get_simulation_path
-from mqt import ddsim
+from typing import Union
```

Comment:
Union shouldn't be needed anymore. You can write any `Union[A, B]` as `A | B`.
Also something that could even be automatically fixed by pre-commit ;o)

Reply:
I tried to make this change and I think I understand the intention, but I find `Union[...]` to be more readable :sweat_smile:

Reply:
I'd still advocate for changing it though as it is the modern way to specify typing information and it does not require an import.
Additionally it renders much nicer in the documentation and has the semantic meaning of "or".

Reply:
Okay

Reply:
Note, however, that it is best practice nowadays to 
```python
from future import __annotations__
```
as the very first line in the Python file. See https://docs.python.org/3/library/__future__.html and in particular https://peps.python.org/pep-0563/

Reply:
> ... it does not require an import...

:neutral_face:

Reply:
Yeah, I know.. still, as stated above, this will become mandatory anyway.
And, at least it is only a single include for all the typing stuff. This also makes things like `Set` and `List` obsolete.
And these `future` imports can also be added automatically by pre-commit.

Reply:
Added the statement.